packages/engine/package.json

```diff
@@ -66,7 +66,7 @@
     "simplex-noise": "^4.0.1",
     "three": "0.157.0",
     "three-mesh-bvh": "^0.6.8",
-    "three.quarks": "^0.10.14",
+    "three.quarks": "^0.10.18",
```

Comment:
now that we have a patch version we should pin this to .18

Reply:
Honestly I am hoping that the maintainer updates the npm package by tomorrow. So I think I'm just gonna wait and revert these changes. I thought it might take them much longer to merge my PR.

Reply:
(If that's alright with you)

Reply:
I did that one character change for now. We never know what the author might have going on.